src/storage/types.ts

```diff
@@ -55,4 +55,24 @@ export interface TeamMemberDocument {
   }>;
   createdAt: Date;
   updatedAt: Date;
+}
+
+export interface SarsCov2EstimateDocument {
```

Comment:
what si this and how is it different from the typedefs and generated gql types etc.

Reply:
Yup! This is the type that is stored in our database whereas the typedefs and generated types are the types returned by the API. The field names and types of the fields are slightly different. The difference is especially clear with the team member types for the database and the typedefs for the API.